_pages/teams.md

```diff
@@ -40,14 +37,15 @@ This will determine the URL where your organization will be showcased. It is req
 * *Organization Full Name:* This is the organization's full name.
 * *Contact Name:* The primary contact for your organization.
 * *Organization Email:* The organization's primary email.
-* *Url:* Your organization's website.
+* *Url:* Your organization's website. This property is optional.
+* *Logo:* You may optionally add an image to brand your organization.
```

Comment:
Does this need to be a public URL?  Or could it be a data:// URI?  We should clarify here.